src/main/java/com/btkelly/gnag/api/GitHubApi.java

```diff
@@ -70,12 +75,24 @@ public Response intercept(Chain chain) throws IOException {
             }
         });
 
+        if (gnagPluginExtension.debugLogEnabled()) {
+            HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
+            interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
Don't mind either way, just checking I wasn't missing something. It would be neater to consolidate, probably, but nbd. LMK what you decide!